zrml/prediction-markets/src/weights.rs

```diff
@@ -10,7 +10,7 @@
 // pallet
 // --chain=dev
 // --steps=10
-// --repeat=2
+// --repeat=1000
```

Comment:
[This is not the case](https://github.com/zeitgeistpm/zeitgeist/blob/v0.3.2/zrml/prediction-markets/src/weights.rs). The weights were changed within a PR and `2` was used because the benchmark workflow does not work properly on forks yet.